tests/functional/backend/corpora/test_revisions.py

```diff
@@ -166,10 +166,11 @@ def test_revision_flow(self):
             res = requests.get(f"{self.api}/dp/v1/datasets/meta?url={self.create_explorer_url(deleted_dataset_id)}")
             self.assertEqual(res.status_code, 404)
 
-            # Endpoint is eventually consistent
-            (final_status_code, desired_status_code) = (None, 404)
-            for i in range(20):
-                res = requests.get(f"{self.api}/cellxgene/e/{original_dataset_id}.cxg/api/v0.2/schema")
+            # Endpoint is eventually consistent. This redirects to the collection page, so the status we want is 302
+            (final_status_code, desired_status_code) = (None, 302)
```

Comment:
Would it be possible to also check the redirect link? 
Also this test is basically checking explorer functionality at this point. Its not a bad thing to be checking but I'm not sure it should be a part of the portal code base

Reply:
Yeah, this was previously checking for a 404. I think checking the redirect link shouldn't belong here, especially because if we change it in the explorer, this will break and the portal doesn't depend on the explorer. I would leave this as is.